docs/releases/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Papers & Releases
+hide:
+  - toc
+---
+
+On this page you can find a list of papers, model and dataset releases that were created using NeMo-Skills.
```

Comment:
Should we add `OpenCodeReasoning` to this as well?

Reply:
Yes, that's the plan. But in another pr